File: src_simple/python/simple/le_plus_grand.py
#!/usr/bin/env python
# -*- coding=utf-8 -*-

""" le_plus_grand  -  11.03.2014

Lire x et y et dire lequel est le plus grand """

# lecture de x et y
x = float(raw_input("entrer x : "))
print("x = {0}".format(x))

y = float(raw_input("entrer y : "))
print("y = {0}".format(y))

# test entre x et y
if x > y:
    print("x est plus grand")
    print("le plus grand = {0}".format(x))
elif y > x:
    print("y est plus grand")
    print("le plus grand = {0}".format(y))
else:
    print("x et y sont égaux")
    print("x = {0}\t y = {1}".format(x, y))

File: src_simple/f90/le_plus_grand.f90
! le_plus_grand  -  11.03.2014

! ******************************************
! Lire x et y et dire lequel est le plus grand
! ******************************************

program le_plus_grand

    implicit none

    real :: x, y
  
    ! lecture de x et y
    write(*, *) "Entrer x : "
    read(*, *) x
    write(*, *) "x = ", x

    write(*, *) "Entrer y : "
    read(*, *) y
    write(*, *) "y = ", y

    ! test entre x et y
    if (x > y) then
        write(*, *) "x est plus grand"
    else if (x < y) then
        write(*, *) "y est plus grand"
    else
        write(*, *) "x et y sont égaux"
    end if
end program le_plus_grand
